.config-schema.json

```diff
@@ -430,6 +430,29 @@
                     "x-env-variable": "OPENFGA_CHECK_QUERY_CACHE_TTL"
                 }
             }
+        },
+        "dispatchThrottling": {
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "enable throttling when request's number of dispatches is high",
+                    "type": "bool",
+                    "default": "false",
+                    "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_ENABLED"
+                },
+                "timeTickerFrequency": {
+                    "description": "defines how frequent dispatch throttling will be evaluated",
```

Comment:
**Slightly pedantic:**The dispatch throttling isn't being evaluated, the deprioritized queue is.

```suggestion
                    "description": "defines the frequency period that the deprioritized throttling queue is evaluated. A higher value will result in more aggressive throttling.",
```

Reply:
taken suggestion